new-docs/content/any/project/changelogs/NEXT.md

```diff
@@ -20,16 +35,24 @@ draft: true
   of instances are spun up and not cleanly reaped by the GC, such as during a
   long-running test process in 'watch' mode.
 
-### Rust
+- The Polar `Variable` type is now exposed in the Node.js library, allowing users to pass unbound variables to `queryRule()` and `isAllowed()`.
+```
+  const oso = new Oso()
+  await oso.loadStr('hello("world"); hello("something else");');
+  for await (const val of oso.queryRule("hello", new Variable("var"))) {
+      console.log(val);
+  }
+  => 
+  Map(1) { 'var' => 'world' }
+  Map(1) { 'var' => 'something else' }
```

Comment:
```suggestion

```js
const oso = new Oso();
await oso.loadStr('hello("world"); hello("something else");');
const query = oso.queryRule("hello", new Variable("var"));
for await (const result of query) {
  console.log(result);
}

=> Map(1) { 'var' => 'world' }
=> Map(1) { 'var' => 'something else' }
```